test/testkey/generate.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
can you add a comment about this generate.sh, how we'll use it and when to run it?

Reply:
:+1: